packages/server/src/fhir/operations/patientsummary.ts

```diff
@@ -287,41 +298,12 @@ export class PatientSummaryBuilder {
       case 'vital-signs':
         this.vitalSigns.push(obs);
         break;
-      case 'imaging':
```

Comment:
Before, we were trying to be clever about putting `Observation` resources into different sections.  It turns out that wasn't correct, and it would make the summary tables a lot more complicated, so I'm pulling back on that.